acceptance/setup/git/pre-suite/030_PuppetMasterSanity.rb

```diff
@@ -1,6 +1,6 @@
 test_name "Puppet Master sanity checks: PID file and SSL dir creation"
 
-pidfile = '/var/lib/puppet/run/master.pid'
+pidfile = '/var/run/puppetlabs/agent/master.pid'
```

Comment:
Also, I'm a bit unfamiliar with Puppet's pidfile management, so how would we distinguish between using the master or agent pidfile? We seem to only currently define one path without distinguishing whether it's a master running or an agent run, and we use that single path in all of our init scripts.